src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -449,56 +448,60 @@ The features of the database page will vary due to the following:
     id="explains-plans"
     title="Explain plans"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
     (See above)
   </Collapser>
   <Collapser
     className="freq-link"
     id="obfuscation"
     title="Obfuscation"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
 
-    Currently, there is not a uniform way in OpenTelemetry to obfuscate sensitive information. Check each instrumented library you are using to see if there is a configuration option for query sanitization.
+    Currently, there is not a uniform way in OpenTelemetry to obfuscate sensitive information. Check each instrumented library you are using to see if there is a configuration option for query sanitization. The opentelemetry-collector could also be customized to obfuscate any telemetry data.
   </Collapser>
 </CollapserGroup>
 
 ## Externals [#externals]
 
-New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 🟡 Limited support
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 Limited support
 
 The external services feature allows you to look closely at the upstream and downstream activity of a single service. Whether you’re monitoring with New Relic agents or OpenTelemetry, the external services feature shows transaction-level detail as you drill into connected services.
 
 For OpenTelemetry, this view is based strictly on span data (which may be sampled), while New Relic agents report unsampled metrics on the interservice connections.
 
 ## Logs [#logs]
 
-New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 🟡 Limited support
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
+
+New Relic agents offer automatic log collection by default for services. OpenTelemetry can be configured to send logs from a variety of sources.
 
 ## Service map [#service-maps]
 
 New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
 
-This map displays an entire set of interconnected services for a given service. The interconnected services must be actively reporting to appear in this map. Compare with these features: related entities and automap.
+The Service Map, accessible when looking at a single service in the UI, displays an entire set of entities connected to that service and is currently only supported by New Relic agents. The interconnected services must be actively reporting to appear in this map. 
```

Comment:
A little more clarity on what "service map" specifically means. More consolidation of the maps content in a future PR.